package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.5",
```

Comment:
![image](https://github.com/EmiLoKiJu/my-app/assets/93039359/6325d757-a6e0-4135-bbc2-b86c9742a819)

**[OPTIONAL]**

- [ ] It would be nice if you replace the title and description with your project title and description to make your readme more professional and descriptive 😊